buildpipeline/osx.groovy

```diff
@@ -0,0 +1,76 @@
+@Library('dotnet-ci') _
+
+// Incoming parameters.  Access with "params.<param name>".
+// Config - Build configuration. Note that we don't using 'Configuration' since it's used
+//          in the build scripts and this can cause problems.
+// OuterLoop - If true, runs outerloop, if false runs just innerloop
+
+def submittedHelixJson = null
+
+simpleNode('OSX10.12','latest') {
```

Comment:
Thanks @mmitche if you get the OSX machines ready I can just forgo using helix for these test runs until you can get the open queues ready.